evap/staff/views.py

```diff
@@ -1323,6 +1323,7 @@ def notify_reward_points(grantings, **_kwargs):
         "editable": editable,
         "questionnaires_with_answers_per_contributor": questionnaires_with_answers_per_contributor,
     }
+    del notify_reward_points  # cleanup receiver
```

Comment:
Why does this solve the problem? From [the docs](https://docs.python.org/3/reference/simple_stmts.html#grammar-token-python-grammar-del_stmt):
> Deletion of a name removes the binding of that name from the local or global namespace, depending on whether the name occurs in a [global](https://docs.python.org/3/reference/simple_stmts.html#global) statement in the same code block.

To me, this sounds like `del x` and `x` going out of scope are the same thing. Both don't ensure that garbage collection actually runs. The tests pass, but that's probably just a CPython implementation detail?

There is [`Signal.disconnect`](https://docs.djangoproject.com/en/5.0/topics/signals/#disconnecting-signals). Ideally I'd want to use that through a context manager. I can't find any matching context manager in the django docs, we'd probably have to make our own, but that should be feasible.

Reply:
my explaination: python uses both gc and ref counting. Maybe going out of scope does not mean directly that the refcount is decreased.
I tested this entirely without gc and test failures were avoidee

Reply:
according to [this stackoverflow](https://stackoverflow.com/a/9063805) you are right: refcounting _is_ part of "gc" and is  even implementation dependent :+1: